test/integration/render-error-on-render-error/with-get-initial-props/pages/_error.js

```diff
@@ -0,0 +1,11 @@
+const Error = ({ message }) => {
```

Comment:
this test suite name is not ideal `render-error-on-render-error`

also from the docs, should I have put this in the `production` suite?